Magento_CustomerBalance/templates/checkout/multishipping/payment.phtml

```diff
@@ -0,0 +1,47 @@
+<?php
+// phpcs:disable Magento2.Templates.ThisInTemplate
+?>
+<?php if ($block->isAllowed()) : ?>
+<div class="checkout multishipping payment customerbalance" id="customerbalance-block">
+    <input type="hidden" name="payment[use_customer_balance]" value="0"/>
+
+    <div class="field choice checkbox">
+        <input
+            type="checkbox"
+            class="checkbox__field"
+            name="payment[use_customer_balance]" value="1"
+            id="use-customer-balance"
+            data-mage-init='{"checkoutBalance":{
+                "amountSubstracted":"<?= /* @noEscape */ $block->isCustomerBalanceUsed() ?>",
+                "usedAmount":"<?= (float)$block->getQuote()->getBaseCustomerBalAmountUsed() ?>",
+                "balance":"<?= (float)$block->getBalance() ?>"
+            }}'
+            <?= ($block->isCustomerBalanceUsed() ? 'checked' : '') ?>
+        />
+        <svg
+            class="
+                checkbox__icon
+                checkbox__icon--checked
+            "
+            role="presentation"
+            focusable="false"
+        >
+            <use href="<?= $block->escapeUrl($block->getViewFileUrl('/images/icons-sprite.svg#checked')) ?>"></use>
+        </svg>
+        <svg
+            class="
+                checkbox__icon
+                checkbox__icon--unchecked
+            "
+            role="presentation"
+            focusable="false"
+        >
+            <use href="<?= $block->escapeUrl($block->getViewFileUrl('/images/icons-sprite.svg#unchecked')) ?>"></use>
+        </svg>
+        <?php /* @noEscape */ $balance = $this->helper(Magento\Framework\Pricing\Helper\Data::class)->currency($block->getBalance()); ?>
+        <label for="use-customer-balance" class="checkbox__label">
```

Comment:
some general rules:
- if an element has more than one attribute, then each attribute should be in a separate line
- if an element has more than 2 classes, then each class should be in a separate line


please adjust